code/main/edge/konductorConfig.brs

```diff
@@ -0,0 +1,52 @@
+' ********************** Copyright 2024 Adobe. All rights reserved. **********************
+' *
+' * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+' * you may not use this file except in compliance with the License. You may obtain a copy
+' * of the License at http://www.apache.org/licenses/LICENSE-2.0
+' *
+' * Unless required by applicable law or agreed to in writing, software distributed under
+' * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' * OF ANY KIND, either express or implied. See the License for the specific language
+' * governing permissions and limitations under the License.
+' *
+' *****************************************************************************************
+
+' ******************************* MODULE: EdgeRequestWorker *******************************
+
+function _adb_KonductorConfig() as object
+
+    return {
+        _stateStore: invalid,
```

Comment:
You should also persist `stateStore` and `locationHint` across app launches.

Reply:
Yes we have been discussing the storage part due to the memory limitations on the channel side. We were thinking to cache this to avoid eating up the persistent storage refer the consent internal wiki. What are your thoughts?